pingdom-checks/benefits-reference-data.ping

```diff
@@ -0,0 +1,34 @@
+# -*- sh -*-
+
+BENEFITS_REFERENCE_DATA_SLACK_ID="${SLACK_VASDVP_MONITORING},${SLACK_API_ALARMS_BENEFITS_SERVICES}"
+BENEFITS_REFERENCE_DATA_BACKEND_SLACK_ID="${SLACK_LIGHTHOUSE_ALERTS},${SLACK_API_ALARMS_BENEFITS_SERVICES}"
+
+#
+# Sandbox
+#
+
+# Benefits Reference Data Healthcheck
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-benefits-reference-data-health \
+  -a host=sandbox-api.va.gov \
+  -a url="/services/benefits-reference-data/healthcheck/liveness" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group="benefits-services" \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$BENEFITS_REFERENCE_DATA_SLACK_ID,$PAGER_DUTY"
+
+# Benefits Reference Data Backend Healthcheck
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-benefits-reference-data-backend-health \
+  -a host=sandbox-api.va.gov \
+  -a url="/services/benefits-reference-data/healthcheck/readiness" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group="benefits-services" \
+  -a userids_csv="$NO_USERS" \
```

Comment:
I believe there's a missing piece here. Normally, the backend healthchecks populate the API's component in StatusPage, so there should be an userid here for the new API. This requires manual intervention as well from a StatusPage admin to create that component in SP and then to create the integration in Pingdom to extract the userid. @bschofield-va is this process documented, or should we work together on this? Ideally since now Leeroy is also a  code owner in the repo, we could assist with the manual bits as well where needed.

Reply:
It sounded like one of Cassowary / Quokka / Marmoset were going to work with the appropriate team to get the status page connection setup, so I left it off for now.